modules/ephemeral-storage-creating-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * work_with_shared_resources/using-shared-resource-csi-driver.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-creating-configmaps-across-namespaces_{context}"]
+= Creating a SharedConfigMap instance
+
+You can create a `SharedConfigMap` CR instance for the `ConfigMap` object that you want to share across namespaces. 
+
+.Prerequisites
+
+You must have permissions to perform the following actions:
+
+* Create `sharedconfigmaps.sharedresource.openshift.io` custom resource definition (CRD) at a cluster-scoped level.
+* Create a `ClusterRole` object for the `SharedConfigMap` CR.
+* Create role and role binding for the Shared Resource Container Storage Interface (CSI) Driver.
+* Manage roles and role bindings across the namespaces in the cluster to control which users can get, list, and watch instances.
+* Manage roles and role bindings to allow the service account specified by a pod to use the `SharedConfigMap` CR to mount your `csi` volume that references a `ConfigMap` object .
```

Comment:
Manage roles and role bindings so that the service account specified by a pod can use the `SharedConfigMap` CR to mount a Container Storage Interface (csi) volume that references a `ConfigMap` object.